app/Models/CategoryTagModel.php

```diff
@@ -9,4 +9,21 @@ class CategoryTagModel extends Model
     protected $table         = 'category_tags';
     protected $primaryKey    = 'categoryTagId';
     protected $allowedFields = ['categoryTagName', 'categoryId'];
+
+    /**
+     * get all the category tags from all categories selected
+     *
+     * @param array $categoriesIds the categories id´s from search
+     *
+     * @return array
+     */
+    public function getAllByCategories(array $categoriesIds = [])
+    {
+        $builder = $this->select(['categoryTagId', 'categoryTagName']);
```

Comment:
Nitpick: Maybe change this to a more specific name like queryBuilder.
```suggestion
        $queryBuilder = $this->select(['categoryTagId', 'categoryTagName']);
```

CC: @FernandoJoachin